include/boost/detail/winapi/jobs.hpp

```diff
@@ -10,8 +10,23 @@
 
 #include <boost/detail/winapi/basic_types.hpp>
 #include <boost/detail/winapi/access_rights.hpp>
+#include <boost/detail/winapi/detail/cast_ptr.hpp>
 
-#if !defined( BOOST_USE_WINDOWS_H )
+// Microsoft documentation states all of these APIs are available
+// in Windows XP / Windows 2003 or later
+
+#if (BOOST_USE_WINAPI_VERSION >= BOOST_WINAPI_VERSION_WINXP)
```

Comment:
I don't think this is correct. Some functions, e.g. `CreateJobObjectA`, are available since Win2k.

Reply:
The CI build proved otherwise, failing when _WIN32_WINNT=0x0500.

This is further proven by the documentation for CreateJobObject at https://msdn.microsoft.com/en-us/library/windows/desktop/ms682409(v=vs.85).aspx which specifically states: 

Minimum supported client | Windows XP [desktop apps only]
-- | --

Reply:
MSDN cannot be trusted as it doesn't mention systems prior to XP. Not consistently, anyway. I can see `CreateJobObjectA` defined in C:\Program Files (x86)\Windows Kits\10\Include\10.0.15063.0\um\WinBase.h:7563, guarded by `#if (_WIN32_WINNT >= 0x0500)` a few lines above.

Reply:
I see that as well, so I will fix this up.